spacy/cli/init_model.py

```diff
@@ -242,7 +243,8 @@ def add_vectors(
         if vectors_data is not None:
             nlp.vocab.vectors = Vectors(data=vectors_data, keys=vector_keys)
     if name is None:
-        nlp.vocab.vectors.name = f"{nlp.meta['lang']}_model.vectors"
+        # TODO: Is this correct? Does this matter?
+        nlp.vocab.vectors.name = f"{nlp.meta['lang']}_{nlp.meta['name']}.vectors"
```

Comment:
@honnibal Does this matter / do we need this?

Reply:
I think it's good to set this, yeah. Otherwise it might be hard to track down vectors later.

Reply:
Yeah, but what about the specific name and how it's generated?